src/pleasant-lcd.h

```diff
@@ -234,6 +236,50 @@ void lcd_fill_rect(uint16_t x,
                    uint16_t h,
                    lcd_color color);
 
+/* Touch ------------------------------------------------------------------- */
+typedef struct {
+  uint32_t x;
+  uint32_t y;
+} CAL_POINT;
+
+typedef struct {
+  uint32_t a;
+  uint32_t b;
+  uint32_t c;
+  uint32_t d;
+  uint32_t e;
+  uint32_t f;
+  uint_least32_t div;
+} CAL_MATRIX;
+
+CAL_MATRIX tp_matrix;
```

Comment:
Move this to the State block below, and make it `extern` like everything there. Then initialize it to a sane value at the top of `pleasant-lcd.c`